boards/arm/cc3220sf_launchxl/doc/cc3220sf_launchxl.rst

```diff
@@ -210,6 +212,41 @@ build target:
    :maybe-skip-config:
    :goals: debug
 
+
+WiFi Support
+************
+
+The SimpleLink Host Driver, imported from the SimpleLink SDK, has been ported to
+Zephyr, and communicates over a dedicated SPI to the network co-processor.
+It is available as a Zephyr WiFi device driver in :file:`drivers/wifi/simplelink`.
+
+Currently only the Zephyr WiFi management operations are exported.
+
+Usage:
+======
+
+Set CONFIG_WIFI_SIMPLELINK=y to enable WiFi.
```

Comment:
It would be better to have a link to the config option, so how about:

    Set :option:`CONFIG_WIFI_SIMPLELINK` to ``y`` to enable WiFi.

Reply:
done.
Thanks for the quick review!